tasks/package.yml

```diff
@@ -11,7 +11,7 @@
 
 - name: Nginx | Make sure nginx is installed (package)
   apt:
-    pkg: nginx
```

Comment:
Agreed. it makes sense to use the nginx ansible var for this purpose. Initially I wasn't sure about the intent.